clients/apps/web/src/components/Notifications/Popover.tsx

```diff
@@ -112,10 +112,6 @@ const Popover = ({ type = 'topbar' }: { type?: 'topbar' | 'dashboard' }) => {
           aria-live="assertive"
           className={notificationsContainerClassnames}
           ref={ref}
-          onClick={(e) => {
-            e.preventDefault()
-            e.stopPropagation()
-          }}
```

Comment:
I wonder why this was here 😕

Reply:
Me neither. I suspect it was needed when we had the notification area in the topbar; but that doesn't really make sense neither.